packages/oui-inline-adder/README.md

```diff
@@ -0,0 +1,159 @@
+                    disabled="!item.isNew">
+                </oui-numeric>
+            </oui-inline-adder-cell>
+        </oui-inline-adder-row>
+    </oui-inline-adder-item>
+</oui-inline-adder>
+```
+
+### Multiple row items
+
+An Inline Adder can have more than one row and be displayed as a group.
+
+```html:preview
+<oui-inline-adder
+    on-add="$ctrl.addItem(item)"
+    on-remove="$ctrl.removeItem(item)">
+    <oui-inline-adder-item data-ng-repeat="item in $ctrl.quadraFieldItems" item="item" is-new-item="item.isNew" aria-add-item="Add new item" aria-remove-item="Remove item {{$index+1}}">
+        <oui-inline-adder-row>
+            <oui-inline-adder-cell>
+                <label for="quadra-field-prop1" class="oui-label">Property 1</label>
+                <input type="text" class="oui-input"
+                    id = "quadra-field-prop1"
+                    data-ng-model="item.prop1"
+                    data-ng-disabled="!item.isNew">
+            </oui-inline-adder-cell>
+            <oui-inline-adder-cell>
+                <label for="quadra-field-prop2" class="oui-label">Property 2</label>
+                <input type="text" class="oui-input"
+                    id = "quadra-field-prop2"
+                    data-ng-model="item.prop2"
+                    data-ng-disabled="!item.isNew">
+            </oui-inline-adder-cell>
+        </oui-inline-adder-row>
+        <oui-inline-adder-row>
+            <oui-inline-adder-cell>
+                <label for="quadra-field-prop3" class="oui-label">Property 3</label>
+                <input type="text" class="oui-input"
+                    id = "quadra-field-prop3"
+                    data-ng-model="item.prop3"
+                    data-ng-disabled="!item.isNew">
+            </oui-inline-adder-cell>
+            <oui-inline-adder-cell>
+                <label for="quadra-field-prop4" class="oui-label">Property 4</label>
+                <input type="text" class="oui-input"
+                    id = "quadra-field-prop4"
+                    data-ng-model="item.prop4"
+                    data-ng-disabled="!item.isNew">
+            </oui-inline-adder-cell>
+        </oui-inline-adder-row>
+    </oui-inline-adder-item>
+</oui-inline-adder>
+```
+
+**Note**: The `on-add` and `on-remove` callbacks receive the item being added or removed. These callbacks should return a promise, which is resolved after a new item is added or after the item is removed respectively.
+
+## API
+
+### oui-inline-adder
+
+| Attribute         | Type            | Binding | One-time binding | Values                    | Default             | Description                        |
+| ----              | ----            | ----    | ----             | ----                      | ----                | ----                               |
+| `on-add`          |                 | &       |                  |                           |                     | new item add handler (button)      |
```

Comment:
Type is "function".
You should mention that the handler must return a promise (because you call `finally` on it).

Reply:
Sorry. I missed that somehow. The documentation is corrected now.